modules/docs/arrow-docs/docs/docs/README.md

```diff
@@ -74,13 +77,16 @@ You can find the dependencies necessary in the Basic Setup of the README at the
 Add the dependencies into the project's `build.gradle`
 
 ```groovy
-def arrow_version = "0.9.0"
+apply plugin: 'kotlin'
+apply plugin: 'kotlin-kapt'
+def arrow_version = "0.9.0-SNAPSHOT"
 dependencies {
     compile "io.arrow-kt:arrow-core-data:$arrow_version"
     compile "io.arrow-kt:arrow-syntax:$arrow_version"
     compile "io.arrow-kt:arrow-typeclasses:$arrow_version"
     compile "io.arrow-kt:arrow-extras:$arrow_version"
     compile "io.arrow-kt:arrow-core-extensions:$arrow_version"
+    compile "io.arrow-kt:arrow-data-extensions:$arrow_version"
```

Comment:
This is no longer available. It's the one below arrow extras extensions, the plugin directives are fine though. Thanks!

Reply:
Should I delete data-core-extension or core-extensions ? 
I am not sure..